docs/user-guide/src/SUMMARY.md

```diff
@@ -24,5 +24,6 @@
         - [Node Reuse]()
         - [Pivoting]()
 - [Ip-address-manager]()
+    - [Introduction](ipam/ipam.md)
```

Comment:
Can you please move this document under `[Ip-address-manager]()`, I guess we don't need a  new section